packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -769,6 +776,21 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
     }
 
     private verifyNotClosed(callSite: string, checkTombstone = true, safeTelemetryProps: ITelemetryProperties = {}) {
+        if (this.deleted) {
+            const messageString = `Context is deleted! Call site [${callSite}]`;
+            const error = new DataCorruptionError(messageString, safeTelemetryProps);
+            this.mc.logger.sendErrorEvent(
```

Comment:
I renamed the `sendGCTombstoneEvent` method and used it for sending sweep / delete logs as well. See https://github.com/microsoft/FluidFramework/blob/301eabdcb43d8b23a80a2e3e440363f8d3edc04b/packages/runtime/container-runtime/src/blobManager.ts#L780 for example.
You could use that too. Also, maybe use `Deteled` not `Sweep` as per our discussion in the comment in my PR?

Reply:
Oh yeah, I forgot to switch this back to Deleted, I'll pull in main here

Reply:
Note: still need to refresh from main